### PR TITLE
improve test coverage of some classes of the time package

### PR DESCRIPTION
**Pull request by PascalSchumacher**

### CODE REVIEW

**Comment by PascalSchumacher** [comment]
The other classes of the package can not be added to `TimeRandomizersTest` until https://github.com/benas/random-beans/issues/105 is resolved.

File: random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/FloatRangeRandomizerTest.java [hunk 1]
```diff
@@ -46,21 +48,31 @@ public void generatedValueShouldBeWithinSpecifiedRange() {
...
-        randomizer = new FloatRangeRandomizer(min, null);
+        randomizer = aNewFloatRangeRandomizer(min, null);
         Float randomFloat = randomizer.getRandomValue();
         assertThat(randomFloat).isGreaterThanOrEqualTo(min);
     }
 
+    @Test
+    public void shouldAlwaysGenerateTheSameValueForTheSameSeed() {
+        // given
+        FloatRangeRandomizer bloatRangeRandomizer = aNewFloatRangeRandomizer(min, max, SEED);
```
Comment (benas) [inline random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/FloatRangeRandomizerTest.java]: I think you mean floatRangeRandomizer here

File: random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/IntegerRangeRandomizerTest.java [hunk 2]
```diff
@@ -46,21 +48,31 @@ public void generatedValueShouldBeWithinSpecifiedRange() {
...
-        randomizer = new IntegerRangeRandomizer(min, null);
+        randomizer = aNewIntegerRangeRandomizer(min, null);
         Integer randomInteger = randomizer.getRandomValue();
         assertThat(randomInteger).isGreaterThanOrEqualTo(min);
     }
 
+    @Test
+    public void shouldAlwaysGenerateTheSameValueForTheSameSeed() {
+        // given
+        IntegerRangeRandomizer bntegerRangeRandomizer = aNewIntegerRangeRandomizer(min, max, SEED);
```
Comment (benas) [inline random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/IntegerRangeRandomizerTest.java]: typo here: bntegerRangeRandomizer -> integerRangeRandomizer

File: random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/StringLengthRandomizerTest.java [hunk 3]
```diff
@@ -48,21 +50,31 @@ public void generatedStringLengthShouldBeWithinSpecifiedRange() {
...
     @Test
     public void whenSpecifiedMaxLengthIsNullThenShouldUseDefaultMaxValue() {
-        randomizer = new StringLengthRandomizer(min, null);
+        randomizer = aNewStringLengthRandomizer(min, null);
         String randomString = randomizer.getRandomValue();
         assertThat(randomString.length()).isGreaterThanOrEqualTo(min);
     }
 
+    @Test
+    public void shouldAlwaysGenerateTheSameValueFtringheSameSeed() {
```
Comment (benas) [inline random-beans/src/test/java/io/github/benas/randombeans/randomizers/range/StringLengthRandomizerTest.java]: typo: shouldAlwaysGenerateTheSameValueFtringheSameSeed -> shouldAlwaysGenerateTheSameValueForTheSameSeed

**Comment by PascalSchumacher** [comment]
Thanks for the review. :)

Sorry for the typos, that's what you get for copy&paste + search&replace. :(

I have updated the pull request.

**Comment by benas** [comment]
Hey you don't have to be sorry!

Thank you for the update

**Pull request merged**